index.html

```diff
@@ -29,45 +61,56 @@ <h2>Telemetry portal</h2>
             <div class="col-md-4">
                 <h3>Main tools</h3>
                 <div class="dashboard-list list-group" style="margin: 0">
-                    <a class="list-group-item" href="new-pipeline/dist.html">
+                    <a class="list-group-item" href="new-pipeline/dist.html"
+                       onclick="trackOutboundLink(this); return false;">
```

Comment:
Could you shorten this to just `return trackOutboundLink(this);` (same for all subsequent)